MailCore/Cache/DraftContentManager.swift

```diff
@@ -313,7 +313,7 @@ public class DraftContentManager: ObservableObject {
 
     private func loadReplyingMessageAndFormat(_ message: Message, replyMode: ReplyMode) async throws -> String {
         let replyingMessage = try await loadReplyingMessage(message, replyMode: replyMode)
-        return Draft.replyingBody(message: replyingMessage, replyMode: replyMode)
+        return await Draft.replyingBody(message: replyingMessage, replyMode: replyMode)
```

Comment:
We need to make sure `replyingMessage` is frozen or detached to keep Realm happy.
If it is, no need to add some extra code.